### PR TITLE
Enable cloud-init without any aid from ds-identify

### PR DESCRIPTION
**Pull request by weizhouapache**
This is for the issue described in https://github.com/apache/cloudstack/issues/8129

----
📚 Documentation preview 📚: https://cloudstack-documentation--383.org.readthedocs.build/en/383/

### CODE REVIEW

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

**Review by DaanHoogland — APPROVED** [review]

File: source/adminguide/templates/_cloud_init.rst [hunk 1]
```diff
@@ -62,6 +62,11 @@ These features can be implemented in `“Linux Template creation process” <_cr
      datasource:
        CloudStack: {}
        None: {}
+
+   .. note::
+      For the vm instances running on VMware or XenServer/XCP-ng hypervisors, if there are multiple cloud-init data sources, it is a known issue that ds-identify is not able to detect if "CloudStack" DataSource is enabled. To fix the problem, please run the following command to enable cloud-init without any aid from ds-identify.
+
+        echo "policy: enabled" >  /etc/cloud/ds-identify.cfg
```
Comment (vishesh92) [inline source/adminguide/templates/_cloud_init.rst:69]:
```suggestion
        .. code:: bash

          echo "policy: enabled" >  /etc/cloud/ds-identify.cfg
```

Comment (weizhouapache) [reply source/adminguide/templates/_cloud_init.rst:69]: thanks @vishesh92 
let's see if it works

Comment (weizhouapache) [reply source/adminguide/templates/_cloud_init.rst]: looks good

image

https://cloudstack-documentation--383.org.readthedocs.build/en/383/adminguide/templates/_cloud_init.html

**Review by vishesh92 — APPROVED** [review]
lgtm

**Pull request merged**